test/express.test.ts

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
this is not a test file - it is a standalone script - only test files should have the name format `foo.test.ts`
rename to test/express-example.ts